daprdocs/config.toml

```diff
@@ -10,6 +10,18 @@ enableGitInfo = true
 # Language Configuration
 languageCode = "en-us"
 
+[languages]
+  [languages.en]
+    title = "English"
+    weight = 1
+    contentDir = "content/en"
+    languageName = "English"
+  [languages.zh-hans]
+    title = "简体中文"
```

Comment:
This should be the translated version "Dapr Docs"

Reply:
done